copier/main.py

```diff
@@ -1114,7 +1188,8 @@ def _apply_update(self) -> None:  # noqa: C901
                         # before storing the file name for marking it as unmerged after the loop.
                         with open(fname) as conflicts_candidate:
                             if any(
-                                line.rstrip() in {"<<<<<<< before updating", ">>>>>>> after updating"}
+                                line.rstrip()
+                                in {"<<<<<<< before updating", ">>>>>>> after updating"}
```

Comment:
There are a few other style changes like this and I wonder if this is because of a discrepancy again between Copier's config and the contributor's editor/IDE configuration.